lib/streamlit/DeltaGenerator.py

```diff
@@ -1909,25 +1912,30 @@ def slider(
         ----------
         label : str or None
             A short label explaining to the user what this slider is for.
-        min_value : int/float or None
+        min_value : int/float/datetime or None
```

Comment:
`datetime` should be `date/time/datetime` :scream: 

Same below

Reply:
Done.